tests/estimators/speech_recognition/test_pytorch_deep_speech.py

```diff
@@ -368,6 +374,10 @@ def _test_all(self, request, setup_class):
             assert not ((transcriptions1 == transcriptions2).all())
 
         else:
+            # Create the optimizer
+            parameters = self.speech_recognizer.model.parameters()
+            self.speech_recognizer._optimizer = torch.optim.SGD(parameters, lr=0.01)
+
             # Before train
             transcriptions1 = self.speech_recognizer.predict(self.x, batch_size=2, transcription_output=True)
```

Comment:
lines  (361 to 366) and (377 to 382) are duplicates they should be moved to before line 360

Reply:
Solved.